src/ffi/orx.ml

```diff
@@ -104,8 +104,8 @@ module Texture = struct
     let width = Ctypes.allocate_n Ctypes.float ~count:1 in
     let height = Ctypes.allocate_n Ctypes.float ~count:1 in
     match get_size texture width height with
-    | Error `Orx -> None
-    | Ok () -> Some (!@width, !@height)
+    | Error `Orx -> Fmt.invalid_arg "Unable to retrieve texture size"
+    | Ok () -> (!@width, !@height)
```

Comment:
If we have a texture we should be able to find its size. Cases where we can't are going to be the exception, so let's use exceptions.